winui/AutoComplete/No-Result-Found.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: No Result Found in WinUI AutoComplete control | Syncfusion
+description: Learn how to use NoResultsFoundTemplate and NoResultsFoundContent in Syncfusion WinUI AutoComplete control.
+platform: winui
+control: SfAutoComplete
+documentation: ug
+---
+
+# No Result Found  in WinUI AutoComplete (SfAutoComplete)
+
+When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the `NoResultsFoundContent` property and  the desire text to be displayed for indicating no results found can be customized using the `NoResultsFoundTemplate` property. 

```

Comment:
@YuvarajMasilamani, change this sentence as

When the entered item is not in the suggestion list, SfAutoComplete displays a text indicating there is no search results found. We can set the desire text to be displayed for indicating no results found with the `NoResultsFoundContent` and `NoResultsFoundTemplate` properties.

Reply:
Updated